kart/point_cloud/tilename_util.py

```diff
@@ -51,7 +51,7 @@ def get_tile_path_pattern(
     parent_pattern = (
         re.escape(parent_path.rstrip("/") + "/") if parent_path is not None else ""
     )
-    tile_pattern = re.escape(tilename) if tilename is not None else r"([^/]+)"
+    tile_pattern = re.escape(tilename) if tilename is not None else r"([^/.][^/]*)"
```

Comment:
why this change?

Reply:
Gdal creates temporary, hidden-ish files sometimes eg
aerial.tif -> ._aerial.tif
I had to make sure the raster WC would ignore such files to get the tests to pass - and I made the point-cloud WC ignore such hidden-ish files too for consistency

Reply:
ok. i'd suggest adding that as a comment here. it's not otherwise clear why this regex ignores files starting with a `.` - it might come back to bite us (esp on windows where those files aren't considered 'hidden' necessarily)

Reply:
Done